### PR TITLE
skeleton of myrecipes-page and create form complete

### PR DESCRIPTION
**Pull request by cchchoi1986**

### CODE REVIEW

File: app/assets/javascripts/angular/controllers/myrecipes_controller.js.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+app.controller('MyRecipesCtrl', ['$scope', '$http', '$routeParams', function($scope, $http, $routeParams){
+
+  $scope.mainNavBar = "<%= asset_path('navBarPartial.html') %>";
+  $scope.mainFooter = "<%= asset_path('footerPartial.html') %>";
+
+  $http.get('/myrecipes.json').success(function(data, status, xhr){
+    $scope.recipes = data.recipes;
+    // $scope.name = data.title;
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/myrecipes_controller.js.erb:8]: Do you still needs these comments?

File: app/assets/javascripts/angular/controllers/myrecipes_controller.js.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    // $scope.instructions = data.instructions;
+
+    // $scope.current_user = data.current_user;
+    
+    console.log(data);
+    // console.log(status);
+    // console.log(xhr);
+  });
+  
+  $scope.isCollapsed1 = true;
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/myrecipes_controller.js.erb]: What is this for? The name isn't obvious enough.

File: app/assets/templates/home.html [hunk 3]
```diff
@@ -3,7 +3,10 @@
 <div class="main-body container">
 
   <div class="jumbo row">
-    <img ng-repeat="recipe in recipes" class="splash-item img-responsive  col-xs-3 col-sm-2 col-lg-1" ng-src="{{ recipe.images[0] }}">
+    <div ng-repeat="recipe in recipes">
+      <img class="splash-item img-responsive  col-xs-3 col-sm-2 col-lg-1" ng-src="{{ recipe.images[0] }}">
```
Comment (h8rry) [inline app/assets/templates/home.html:7]: Can you explain where `recipe.images[0]` and `recipe.images[1]` come from? How come we have 2 different images?

**Comment by h8rry** [comment]
The rest of the front-end code looks good

File: app/assets/javascripts/angular/controllers/myrecipes_controller.js.erb (hunk 2 above)
Comment (cchchoi1986) [inline app/assets/javascripts/angular/controllers/myrecipes_controller.js.erb]: will change collaspe filter name

File: app/assets/templates/home.html (hunk 3 above)
Comment (cchchoi1986) [inline app/assets/templates/home.html:7]: 10 image urls per recipe data file

**Pull request merged**